.circleci/config.yml

```diff
@@ -227,6 +246,30 @@ jobs:
       - swissknife/fail_if_dirty:
           print-modified-files: true
 
+  prepare-next-version:
+    description: "Creates a PR with the new snapshot version"
+    docker:
+      - image: cirrusci/flutter
```

Comment:
Does this need to be `cirrusci/flutter` or can it be just a ruby image? I wonder which one is smaller, but I assume the plain ruby one

Reply:
Probably should work with a plain ruby image. Will try it out, we might have the same issue with the cache we had in iOS (or maybe not since it's also a linux image).